 <root BTCPP_format="4" >
     <BehaviorTree ID="MainTree">
        <Sequence name="root_sequence">
            <RobotInitializer   name="RobotInitializer" message="RobotInitializer"/>
        </Sequence>
     </BehaviorTree>
 </root>